extensions/KHR/SPV_KHR_cooperative_matrix.asciidoc

```diff
@@ -415,7 +415,10 @@ be dynamically uniform within the 'Scope' of the cooperative matrix type.
 Linear-algebraic matrix multiply of 'A' by 'B' and then component-wise
 add 'C'. The order of the operations is implementation-dependent. The
 internal precision of floating-point operations is defined by the client
-API. Integer operations used in the multiplication of 'A' by 'B' are
+API. If any of the *Matrix{A,B,C}SignedComponents* operands are present,
```

Comment:
Is the suffix "KHR" missing ?

Reply:
I copied this from somewhere else, I'll fix all the instances of it.